x/dsmr/node_test.go

```diff
@@ -1313,6 +1455,29 @@ func TestAccept_RequestReferencedChunks(t *testing.T) {
 	<-done
 }
 
+func getSignerBitSet(t *testing.T, pChain validators.State, nodeIDs ...ids.NodeID) set.Bits {
```

Comment:
This is super nice, would be great to add this as a convenience function to AvalancheGo